### DISCUSSION TITLE
Is there a point beyond the technical exercice?

### DISCUSSION THREAD

**Discussion by grondilu** [copy/v86#973] [General]
Hi.  I'm interested to know what you guys see think of the purpose for this project.

I used to think that emulation was a path to portability, and in that sense a browser is a very good place to emulate software since browsers are ubiquitous and standardized.  Also, security, as some form of absolute sandboxing or something.

Is it still the idea?  Or is it just for the fun of it?

**Comment by spetterman66** [comment]
For fun! 🙂

It's pretty fun if you develop an emulator for a few years. v86 has first started at emulating Windows 1.0, then 95/98, then XP. As soon as SSE1 and SSE2 support was added, it was able to run every 32-bit Linux distribution and every Windows version up to Windows 8 build 8133. Now we have SSE3 support in v86, thus we're able to run Android-x86 up to 4.4-r3. Emulating a tablet inside a tablet browser isn't a dream anymore, it's actually the reality now. The current version in 2024 differs a lot from the original in 2013. The world of web emulation has also improved a lot beyond v86, you can run classic Macintosh operating systems over at https://infinitemac.org, and alongside classic Mac OS, they're even working at emulating Mac OS X 10.2+.
________________________________
From: Lucien Grondin ***@***.***>
Sent: Friday, January 5, 2024 11:48 AM
To: copy/v86 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [copy/v86] Is there a point beyond the technical exercice? (Discussion #973)


Hi. I'm interested to know what you guys see think of the purpose for this project.

I used to think that emulation was a path to portability, and in that sense a browser is a very good place to emulate software since browsers are ubiquitous and standardized. Also, security, as some form of absolute sandboxing or something.

Is it still the idea?

**Comment by KoghaandSooga** [comment]
How do I do a hard disk as I tried to import my PCEM one and even made a blank PCEM one with nothing on it then transferred it that way on VHD format but it just causes the web emulator to hang on startup where as on PCEM it boots just fine and you do the regular install.

**Comment by spetterman66** [comment]
Virtual x86 does not support the VHD file format. Create a new disk image using QEMU in the IMG format to use it with Virtual x86.
________________________________
